dist/nginx.conf

```diff
@@ -52,21 +52,21 @@ server {
   gzip_http_version 1.1;
   gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript image/svg+xml image/x-icon;
 
-  add_header Strict-Transport-Security "max-age=31536000" always;
+  # add_header Strict-Transport-Security "max-age=31536000" always;
 
   location / {
     try_files $uri @proxy;
   }
 
   location ~ ^/(emoji|packs|system/accounts/avatars|system/media_attachments/files) {
     add_header Cache-Control "public, max-age=31536000, immutable";
-    add_header Strict-Transport-Security "max-age=31536000" always;
+    # add_header Strict-Transport-Security "max-age=31536000" always;
     try_files $uri @proxy;
   }
 
   location /sw.js {
     add_header Cache-Control "public, max-age=0";
-    add_header Strict-Transport-Security "max-age=31536000" always;
+    # add_header Strict-Transport-Security "max-age=31536000" always;
```

Comment:
This should stay too. This isn't served by Rails.

Reply:
No need here either.
Current result:
```
curl --head https://_my_domain_/sw.js

HTTP/2 200
server: nginx/1.23.1
date: Wed, 24 Aug 2022 04:57:39 GMT
content-type: application/javascript
content-length: 181407
vary: Accept-Encoding
etag: "r9fsg01r0gjvqi87ivy7r3b06k3l91ij"
cache-control: public, max-age=0
strict-transport-security: max-age=31536000
alt-svc: h3=":443"; ma=86400
accept-ranges: bytes
```